seed/tests/test_account_views.py

```diff
@@ -98,13 +97,18 @@ def test_dict_org(self):
             'at_organization_token': '',
             'audit_template_user': '',
             'audit_template_password': '',
-            'at_host_url': 'https://api.labworks.org',
             'salesforce_enabled': False,
             'ubid_threshold': 1
         }
 
         org_payload = _dict_org(self.fake_request, [self.org])
+
```

Comment:
if in your `local_untracked` you have set your own better and AT URLs, then this test fails locally. So let's just check to make sure that it matches a regex.